webrtc.gni

```diff
@@ -83,7 +84,7 @@ declare_args() {
 
   # Used to specify an external OpenSSL include path when not compiling the
   # library that comes with WebRTC (i.e. rtc_build_ssl == 0).
-  rtc_ssl_root = ""
+  rtc_ssl_root = woogeen_openssl_header_root
```

Comment:
update to use owt_openssl_header_root instead.

Reply:
I'd like to separate these two changes. This PR only fixes OpenSSL issue, another PR will rename woogeen in GN args.